net-dns/https_dns_proxy/https_dns_proxy-0_pre20200925.ebuild

```diff
@@ -20,12 +20,23 @@ HOMEPAGE="https://github.com/aarond10/https_dns_proxy"
 LICENSE="MIT"
 SLOT="0"
 
-DEPEND="dev-libs/libev
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="dev-libs/libev
```

Comment:
rename `COMMON_DEPEND` to `RDEPEND` and include that in `DEPEND`

Reply:
Done